lib/cli/commands/getorders.ts

```diff
@@ -2,12 +2,21 @@ import callback from '../command';
 import XUClient from '../../xuclient/XUClient';
 import { Arguments } from 'yargs';
 
-export const command = 'getorders';
+export const command = 'getorders [max_results] [pair_id]';
```

Comment:
Let's put `[pair_id]` first here. I see it being more useful to be able to specify a pair but leave the `max_results` as default, for example `getorders BTCLTC`. Would also need to update the example in the README.